jenkins/spark-tests.sh

```diff
@@ -70,10 +70,11 @@ MORTGAGE_SPARK_SUBMIT_ARGS=" --conf spark.plugins=com.nvidia.spark.SQLPlugin \
     --class com.nvidia.spark.rapids.tests.mortgage.Main \
     $RAPIDS_TEST_JAR"
 
-# need to disable pooling for udf test to prevent cudaErrorMemoryAllocation
-CUDF_UDF_TEST_ARGS="--conf spark.rapids.python.memory.gpu.pooling.enabled=false \
-    --conf spark.rapids.memory.gpu.pooling.enabled=false \
-    --conf spark.executorEnv.PYTHONPATH=rapids-4-spark_2.12-0.3.0-SNAPSHOT.jar \
+RAPIDS_FILE_NAME=${RAPIDS_PLUGIN_JAR##*/}
+CUDF_UDF_TEST_ARGS="--conf spark.rapids.memory.gpu.allocFraction=0.1 \
+    --conf spark.rapids.python.memory.gpu.allocFraction=0.1 \
+    --conf spark.rapids.python.concurrentPythonWorkers=2 \
+    --conf spark.executorEnv.PYTHONPATH=${RAPIDS_FILE_NAME} \
```

Comment:
I remember that we failed running IT on blossom due to bad default python, and add below to force use conda one resolved that,
```bash
    --conf spark.driverEnv.PYSPARK_PYTHON="/opt/conda/bin/python" \
    --conf spark.executorEnv.PYSPARK_PYTHON="/opt/conda/bin/python" \
```

Is that still a thing? @firestarman @shotai  @NvTimLiu

Reply:
no, `conda/bin` is added to the head of env `PATH`, so the conda python will be used as the defaut one in IT docker container.

Reply:
hmm, this was still failing my IT on blossom. add two conf flags resolved it though

Reply:
It may be related to the user to launch the Docker container.

Reply:
this sounds like a problem to me. 
People should be able to build the docker image w/ default user and run it with another user.

Does this get doc somewhere?

Reply:
talked offline, this will be included in my blossom migration PR